cli/packages/prisma-client-lib/src/codegen/typescript-client.ts

```diff
@@ -548,7 +546,7 @@ export const prisma = new Prisma()`
     }
 
     return this.renderInterfaceWrapper(
-      `${type.name}${node ? '' : ''}`,
```

Comment:
Someone [forgot this here](https://github.com/prisma/prisma/commit/630ac8930f1348566a1dc4b4b277dabaf42bc4a6#diff-d1ac2c5edb46cae3a12f4ed0eef31d66).